zowe/vscode-extension-for-zowe
# Create Project non-User config when no config exists

## Proposed changes

Resolves #2684

**How to Test**
In an environment with no team config file, creating a new team config file in ZE and selecting "Project" should create a Project Config (zowe.config.json) in the current project directory.
If a Project User Config (zowe.config.user.json) already exists, then ZE should detect that it exists and open that file.

## Release Notes




Milestone: v3

Changelog: Fixed default behavior of "Create a new Team Configuration File" to create a Project Config instead of Project User Config

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_


## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_


## Further comments
